lazyflow/__init__.py

```diff
@@ -32,3 +35,19 @@
 from . import graph
 from . import slot
 from . import operators
+
+
+class N5JsonEncoder(json.JSONEncoder):
+    """
+    json encoder for json dumps in z5py
+    """
+
+    def default(self, obj):
+        if isinstance(obj, np.integer):
+            return int(obj)
+        if isinstance(obj, np.floating):
+            return float(obj)
+        json.JSONEncoder.default(self, obj)
```

Comment:
```suggestion
        return json.JSONEncoder.default(self, obj)
```

Reply:
nice catch

Reply:
Even better:
```suggestion
        return super().default(obj)
```
Although it doesn't really make a difference (this raises an exception).